frontend/src/concepts/k8s/NameDescriptionField.tsx

```diff
@@ -31,6 +31,10 @@ type NameDescriptionFieldProps = {
   onNameChange?: (value: string) => void;
 };
 
+/**
+ * Don't use for K8s name anymore -- that functionality is deprecated.
+ * @see K8sNameDescriptionField
+ */
```

Comment:
Can you add `@deprecated`

Reply:
Eeeeeh... not sure it is tbh. Pipelines needs something. And I want to convert this into that and move it out of the k8s folder when we get there. But until we migrate everything away (which may be a followup I do) we're going to suffer from the weirdness of `k8sName` in the `NameDesc` type living on and making it confusing to know which one to use. Hence the comment.